core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -330,11 +363,15 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
     /**
      * When the user drop a palette item in the dropzone
      *
-     * @param {ActionPayload} event
+     * @param {PositionPayload} positionPayload
      * @return {*}  {void}
      * @memberof EditEmaEditorComponent
      */
-    onPlaceItem(payload: ActionPayload): void {
+    onPlaceItem(positionPayload: PositionPayload): void {
+        this.headlessData.set(positionPayload);
+
+        const payload = this.actionPayload();
```

Comment:
```suggestion
        const payload = this.getPageSavePayload();
```

Set and then get is too cumbersome.